x-pack/solutions/security/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_overview_tab.tsx

```diff
@@ -114,22 +115,23 @@ const VectorScore = ({
     version: string;
   };
 }) => {
+  const { euiTheme } = useEuiTheme();
   const { score, vector, version } = vectorScore;
   return (
     <>
       <EuiFlexGroup
         alignItems="center"
         css={css`
-          background: ${euiThemeVars.euiColorLightestShade};
-          padding: ${euiThemeVars.euiSizeXS} ${euiThemeVars.euiSizeS};
+          background: ${euiTheme.colors.backgroundLightText};
```

Comment:
as shade colors should be replaced and shouldn't be used for background, replacing with more suitable token. But the color is slightly different